hikari/errors.py

```diff
@@ -308,7 +308,27 @@ def __str__(self) -> str:
 
         value = super().__str__()
         if self.errors:
-            value += "\n" + data_binding.dump_json(self.errors, indent=2)
+            value += "\n\n"
+
+            def dump(obj: data_binding.JSONObject, obj_string: str = "") -> str:
```

Comment:
Does this function have to be defined inline and can thus logic just be merged into the parent function?

Reply:
Extracted the logic